javascript/organizing_your_javascript_code/es6_modules.md

```diff
@@ -83,6 +83,7 @@ export { functionOne };
 
 ```javascript
 // another JS file
+// Note: Only use single quote (') or double quote (") for import statements 
```

Comment:
Possibly could start with "You can only use..." To make it explicit it's not just "you should as best practice" but actually "you cannot use template strings for the import"?
Only a small nit but I think it'll be clearer

Reply:
@MaoShizhong just made the change. Let me know what you think.